### ISSUE TITLE
Multiply headers in request

### ISSUE THREAD

**Issue by patryk-marcin-zielinski** [mkotsur/restito#68]
Matching multiple headers not working
```
StubHttp.whenHttp(stubServer)
                .match(
                        Condition.get("/test"),
                        withHeader("Authorization", "Bearer 123233")
                )
```
when request has 2 same header with "Authorization" key, it won't fail.

Problem is in Call class
```
        for (String s : request.getHeaderNames()) {
            call.headers.put(s, request.getHeader(s));
        }
```
`request.getHeader` will return arbitrary one header so even with `Condition.custom` we won't be able to hadle the case with multiply same headers

**Comment by mkotsur** [comment]
Thanks for reporting and figuring this out. I'll have time to work on a fix in the middle of the next week. Of course, if you like to submit a PR, you are more than welcome :-)

**Comment by mkotsur** [comment]
Fixed in https://github.com/mkotsur/restito/pull/69 and available with [0.9.3](https://github.com/mkotsur/restito/blob/master/releases.md) release.

@patryk-marcin-zielinski thanks again!
